ui/app/routes/optimization/supervised-fine-tuning/MetricSelector.tsx

```diff
@@ -41,26 +75,44 @@ export function MetricSelector({
                   field.onChange(metricValue);
                 }}
                 value={field.value ?? "none"}
+                disabled={!functionValue || isLoading}
               >
                 <SelectTrigger>
-                  <SelectValue placeholder="Select a metric" />
+                  <SelectValue
+                    placeholder={
+                      isLoading ? "Loading metrics..." : "Select a metric"
+                    }
+                  />
                 </SelectTrigger>
                 <SelectContent>
                   <SelectItem value="none">
                     <div className="flex w-full items-center justify-between">
                       <span>None</span>
                     </div>
                   </SelectItem>
-                  {Object.entries(config.metrics).map(([name, metric]) => (
-                    <SelectItem key={name} value={name}>
-                      <div className="flex w-full items-center justify-between">
-                        <span>{name}</span>
-                        <div className="ml-2">
-                          <MetricBadges metric={metric} />
-                        </div>
-                      </div>
-                    </SelectItem>
-                  ))}
+                  {Object.entries(config.metrics)
+                    .filter(([name]) => validMetrics.has(name))
+                    .map(([name, metric]) => {
+                      const metricFeedback = metricsFetcher.data?.metrics.find(
+                        (m) => m.metric_name === name,
+                      );
+
+                      return (
+                        <SelectItem key={name} value={name}>
+                          <div className="flex w-full items-center justify-between">
+                            <span>{name}</span>
+                            <div className="ml-2 flex items-center gap-2">
+                              {metricFeedback && (
+                                <span className="text-sm text-muted-foreground">
+                                  ({metricFeedback.feedback_count})
+                                </span>
+                              )}
```

Comment:
can you make this a gray badge that says `Count: {metricFeedback.feedback_count}`?